cluster/cluster.sh

```diff
@@ -0,0 +1,54 @@
+# Copyright 2018-2019 Red Hat, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+export KUBEVIRT_PROVIDER=${KUBEVIRT_PROVIDER:-'k8s-1.17'}
+
+KUBEVIRTCI_VERSION='0b941ea5dc647d3aea6f6a6fff95563d6ce0445e'
+KUBEVIRTCI_REPO='https://github.com/kubevirt/kubevirtci.git'
+# KUBEVIRTCI_PATH var shoud be used only in vendored _kubecirtci folder
```

Comment:
You are talking about KUBEVIRTCI_PATH, but I don't see such thing around

Reply:
well yes, you asked me to remove them entirely from cluster folder. :) 
I left this comment for so that future us might avoid using it again..

Reply:
Documenting all things we should not do would get out of hand pretty fast. You see that this one got out of context for me even before we merged it... I'd prefer to drop it and repeat our conversation when we attempt to use this variable here in the future again.

Reply:
Done